daemon/src/common/Command.cpp

```diff
@@ -208,7 +208,7 @@ namespace Cmd {
 
             //Ignore everything else in an inline comment
             if (inInlineComment) {
-                in++;
+                ++in;
```

Comment:
`in` is an iterator, so it might if std::string doesn't implement iterators as pointers.